error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1166,6 +1167,23 @@ Flux<T> after(Flux<T> flux, Predicate<? super T> predicate, Comparator<? super T
     }
   }
 
+  /**
+   * Prefer {@link Flux#collect(Collector)} with {@link ImmutableList#toImmutableList} over {@link
+   * Flux#collectList()}.
+   */
+  static final class FluxCollectListToCollectImmutableList<T> {
```

Comment:
Name might be a bit verbose, open to suggestions

Reply:
Maybe just `FluxCollectToImmutableList`?

Reply:
Yes that is the perfect suggestions. We base the name on the content of the after template :). Will push the suggestion.